### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: src/main/java/CustomRunnable.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    private static final Logger LOGGER = Logger.getLogger(CustomRunnable.class);
+    private Counter counter;
+
+    public CustomRunnable(Counter counter) {
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        while (counter.getCount() < 100) {
```
Comment (AntonPanchishko) [inline src/main/java/CustomRunnable.java]: Don't use magic number.

File: src/main/java/CustomThread.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    private static final Logger LOGGER = Logger.getLogger(CustomThread.class);
+    private Counter counter;
+
+    public CustomThread(Counter counter) {
+        this.counter = counter;
+    }
+
+    @Override
+    public void run() {
+        while (counter.getCount() < 100) {
```
Comment (AntonPanchishko) [inline src/main/java/CustomThread.java]: same (MN)

File: src/main/java/CustomRunnable.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            counter.setCount(counter.getCount() + 1);
```
Comment (AntonPanchishko) [inline src/main/java/CustomRunnable.java]: Better use for it local variable

File: src/main/java/CustomThread.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            counter.setCount(counter.getCount() + 1);
```
Comment (AntonPanchishko) [inline src/main/java/CustomThread.java]: same(LV)

**Review by AntonPanchishko — CHANGES_REQUESTED** [review]
Good job! Small comments

**Review by AntonPanchishko — APPROVED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**